src/test/temp.rs

```diff
@@ -0,0 +1,51 @@
+use std::time::Duration;
```

Comment:
this appears to be the same test as above, I think it got accidentally committed.

Reply:
Oops, good catch